src/store/validation/index.ts

```diff
@@ -353,6 +354,7 @@ export const validationActions: ActionTree<IValidationState, IRootState> = {
       ?.map(error => extractResourceVariantIndex(error.dataPath))
       .filter(isNotUndefined)
       .map(invalidVariantIndex => resource.values[invalidVariantIndex])
+      .filter(invalidVariant => invalidVariant.mime_type !== getters.choiceMimeType)
```

Comment:
This removes invalid choice resources from 'invalid resources' - we don't care about them on the resource viewer page.

It was giving a console error:
<img width="504" alt="Screenshot 2023-02-15 at 10 25 36 PM" src="https://user-images.githubusercontent.com/18130716/219175983-92014b69-c952-4c9b-8348-1eb4e130d622.png">
when there were empty choices in the middle.